src/components/dashboard/SendLinkSummary.js

```diff
@@ -100,6 +101,78 @@ const SendLinkSummary = ({ screenProps, styles }: AmountProps) => {
   }, [shared])
 
   const handleConfirm = useCallback(() => {
+    const action = screenState.params.action
```

Comment:
destructure action from screenState.params and use it in deps

please, always try to make memoization deps most exact for maximize theirs effect

Reply:
done